src/Microsoft.Diagnostics.Runtime.Utilities/DbgEng/Wrappers/IDebugAdvancedWrapper.cs

```diff
@@ -17,6 +19,42 @@ int IDebugAdvanced.GetThreadContext(Span<byte> buffer)
                 return vtable->GetThreadContext(self, ptr, buffer.Length);
         }
 
+        int IDebugAdvanced.GetCommentWide(out string? comment)
+        {
+            GetVTable(this, out nint self, out IDebugAdvancedVtable* vtable);
+            comment = null;
+            char[] buffer = ArrayPool<char>.Shared.Rent(1048576/2);
```

Comment:
What is this number?  Is it meaningful?  Either way it should be pulled into a constant if this code is left as-is.

Usually, these methods follow a common patern where you can call "Request" and set `&minidumpStream` parameter to `null`.  This tells you the size.  Then allocate a buffer and pass the correct size.  Does that work for this method?

Reply:
Number added to a constant, no specific significance - 1 mb in bytes.
I cannot seem to get the request method to return the size.
int size = 0;
HResult hr = vtable->Request(self, (uint)DEBUG_REQUEST.READ_USER_MINIDUMP_STREAM, null, 0, null, 0, &size);
returns E_INVALIDARG.
I assume that for this case the request method needs the steam type.